etc/eslint/no-bigint-usage/test/no_calls_to_BigInt_methods.test.js

```diff
@@ -0,0 +1,31 @@
+const { rules } = require('../index');
+const { RuleTester } = require('eslint');
+
+const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 2020 } });
+
+ruleTester.run('no-calls-to-BigInt-methods', rules['no-calls-to-BigInt-methods'], {
+  valid: [
+    {
+      name: 'Accepts calls to non-top-level BigInt modules',
+      code: 'custom.BigInt(10);'
+    },
+  ],
+  invalid: [
+    {
+      name: 'Rejects use of BigInt constructor',
+      code: 'let x = BigInt(10)',
+      errors: 1
+    },
+    {
+      name: 'Rejects calls BigInt static methods',
+      code: `
+      let x = BigInt.asIntN(64, 10n);
+      x = BigInt.asUintN(64, 10n);
+      x = BigInt.toString(10n);
+      x = BigInt.valueOf(10n);
+      x = BigInt.whatever();
```

Comment:
```suggestion
      x = BigInt.toLocaleString();
```

We may as well test all 5 static bigint methods if we're going to test 4.

Reply:
Testing for this case would be somewhat different as `BigInt.toLocaleString` [is an instance method ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/toLocaleString) and we'd have to know beforehand that the value the method is being called is a bigint. 

This should be possible, but would require a refactor of the plugin to make use of ts-parser, which I could look more into.